shell/startupfile.go

```diff
@@ -15,23 +15,23 @@ import (
 
 const (
 	beginBlockComment = "# nostromo [section begin]"
-	endBlockComment   = "# nostromo [section end]"
-	sourceCompletion  = "eval \"$(nostromo completion)\""
+	endBlockComment		= "# nostromo [section end]"
+	sourceCompletion	= "eval \"$(nostromo completion)\""
 )
 
 var (
-	startupFilenames   = []string{".profile", ".bash_profile", ".bashrc", ".zshrc"}
-	preferredFilenames = []string{".bashrc", ".zshrc"}
+	startupFilenames = []string{".profile", ".bash_profile", ".bashrc", ".zshrc"}
+	preferredFilenames	 = []string{".bashrc", ".zshrc"}
 )
 
 type startupFile struct {
-	path           string
-	mode           os.FileMode
-	content        string
+	path					 string
+	mode					 os.FileMode
+	content				 string
 	updatedContent string
```

Comment:
Please run `go fmt ./...` if you haven't. It should ensure all the files keep the same formatting.

Reply:
Sorry, haven't set up this computer with all my usual go tooling. Should be updated now